src/components/SearchFilterList.jsx

```diff
@@ -11,11 +11,11 @@ export default function SearchFilterList({
   return (
     <div style={{ margin: '16px 0 0 16px' }}>
       <Text variant="subtitle1" id="ACTIVE_FILTERS" />
-      {formFilters.length === 0 && (
+      {formFilters?.length === 0 && (
         <Text variant="body2" id="NO_ACTIVE_FILTERS" />
       )}
       <Grid container spacing={1} style={{ marginTop: 4 }}>
-        {formFilters.map(filter => (
+        {formFilters?.map(filter => (
```

Comment:
didnt know you could do this, neat!

Reply:
I mean, it didn't error out and it fixed a bug that I caught lol...